### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by sanjanadhikari1**
- [ ] install tailwindcss on the project using Cris's code.
- [ ] create pixel perfect header component
- [ ] make it responsive using tailwindcss
- [ ] Fix 2 issues mentioned.

**FreedCamp**
[https://freedcamp.com/view/2956183/tasks/40068433]()

### CODE REVIEW

File: src/features/header/HeaderScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+  )
+}
+
+const SearhBar = () => {
+  const inputRef = useRef<HTMLInputElement>(null)
+  let placeholder = 'Search creator or NFT'
+  if (inputRef && inputRef.current) {
+    if (inputRef.current.clientWidth < 200) {
+      placeholder = 'Search creator ...'
+    }
```
Comment (CrazyOptimist) [inline src/features/header/HeaderScreen.tsx:61]: This didn't work for me.

**Review by CrazyOptimist — CHANGES_REQUESTED** [review]
### When the screen width reaches to 980px
* top menu link group is not aligned at middle vertically.
* "new NFT" label is wrapped into 2 lines.
* the search input and the question icon sticks too dense

### My opinion
* when screen width reaches to 980, shrink the search field width and truncate the placeholder to "Search ..."
* and the above issues should be resolved

Note that the lower screen width we need to support is 900px, @Dicklesworthstone told us.

File: src/features/header/HeaderScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+        <div className='ml-13 2xl:ml-68'>
+          <SearhBar />
+        </div>
+      </div>
+      <div className='flex items-center mr-33'>
+        {icons.map(icon => (
+          <div className='mr-26'>
+            <Icon src={icon.src} variant={icon.variant} />
+          </div>
+        ))}
```
Comment (CrazyOptimist) [inline src/features/header/HeaderScreen.tsx]: Missing `key` here.

**Review by CrazyOptimist — CHANGES_REQUESTED** [review]

**Review by CrazyOptimist — APPROVED** [review]
Looks good to me 👍

**Review by reacteer — APPROVED** [review]

**Pull request merged**